src/index.ts

```diff
@@ -0,0 +1,235 @@
+/**
+ * Type for (x, y) coordinate pair.
+ */
+ export type Point = { x: number, y: number };
+
+ /**
+  * Method signature for distance calculation.
+  */
+ export type DistanceFunc = (p: Point, a: Point, b: Point) => number;
+ 
+ /**
+  * Distance and index as returned by `MaxDistance()`.
+  */
+ type DistanceIndex = { distance: number, index: number };
+ 
+ /**
+  * Calculates the perpendicular distance between point `p` and the line intersecting points `a` and `b`.
```

Comment:
Codacy has a fix for the issue: [Delete `·`](https://app.codacy.com/gh/icooper/curvereduce-js/pullRequest?prid=9960097)
```suggestion
 * Calculates the perpendicular distance between point `p` and the line intersecting points `a` and `b`.
```